README.md

```diff
@@ -19,19 +16,39 @@ access to their HTTP workloads in Kubernetes by easy integration offered via
 For DevOps and SRE teams, providing ingress for charms that support a relation
 to this charm will be possible via a simple `juju relate` command.
 
-## Project and community
+## Get started
+
+Refer to the [Nginx tutorial](https://charmhub.io/nginx-ingress-integrator/docs/getting-started) for more details on getting started.
+
+### Basic operations
+
+#### Secure an Ingress with TLS
+Refer to [How to secure an Ingress with TLS](https://charmhub.io/nginx-ingress-integrator/docs/secure-an-ingress-with-tls) for step-by-step instructions.
+
+#### Add the Ingress relation
+Refer to [How to add the Ingress relation](https://charmhub.io/nginx-ingress-integrator/docs/add-the-ingress-relation) for step-by-step instructions.
 
-The Nginx Ingress Integrator Operator is a member of the Ubuntu family. It's an
-open source project that warmly welcomes community projects, contributions,
-suggestions, fixes and constructive feedback.
-* [Code of conduct](https://ubuntu.com/community/code-of-conduct)
-* [Get support](https://discourse.charmhub.io/)
-* [Join our online chat](https://chat.charmhub.io/charmhub/channels/charm-dev)
-* [Contribute](https://charmhub.io/nginx-ingress-integrator/docs/contributing)
-* [Roadmap](https://charmhub.io/nginx-ingress-integrator/docs/roadmap)
-Thinking about using the Nginx Ingress Integrator for your next project? [Get in touch](https://chat.charmhub.io/charmhub/channels/charm-dev)!
 
----
+## Integrations
 
-For further details,
-[see the charm's detailed documentation](https://charmhub.io/nginx-ingress-integrator/docs).
+- [ingress](https://charmhub.io/nginx-ingress-integrator/integrations#ingress): Ingress interface that allows to
+  provide an Ingress for the charms supporting this interface. Charms can get an Ingress using the `ingress` or the
+  `nginx-route` interfaces.
+- [nginx-route](https://charmhub.io/nginx-ingress-integrator/integrations#nginx-route): Ingress interface that allows to
+  provide an Ingress for the charms supporting this interface. Charms can get an Ingress using the `ingress` or the
+  `nginx-route` interfaces.
+- [certificates](https://charmhub.io/nginx-ingress-integrator/integrations#certificates): Requires `tls-certificates`
+  interface that facilitates the use of a TLS certificate.
+
+Refer to [Integrations](https://charmhub.io/nginx-ingress-integrator/integrations/) for more information
+about integrations.
+
+## Learn more
+* [Read more](https://charmhub.io/nginx-ingress-integrator)
+* [Official webpage](https://github.com/nginx)
```

Comment:
I am not sure if we should add a link to nginx official page here. Maybe change "Official webpage" to "Learn more about Nginx Ingress"?

Reply:
I will remove it. 

In reality a probably better reference is the kubernetes ingress https://kubernetes.io/docs/concepts/services-networking/ingress/... but is also difficult to explain...